Source/Csla/Core/ContextDictionary.cs

```diff
@@ -98,8 +98,8 @@ public void Add(object key, object value)
     /// <inheritdoc cref="System.Collections.IDictionary.Remove(object)"/>
     public void Remove(object key)
     {
-      object dummy;
-      var removed = TryRemove(key, out dummy);
+      object _;
```

Comment:
I'm not sure you actually need to declare this?

Or maybe you do and it can be

`var removed = TryRemove(key, out var _);`

?

Reply:
Unit tests indicated that syntax is fine as we never use the out value. Changed.